downstream/modules/automation-hub/con-offline-token-active.adoc

```diff
@@ -0,0 +1,11 @@
+
+[id="con-offline-token-active"]
+
+= Keeping your offline token active
+
+Your offline token will never expire by default and is not the subject of SSO Session Idle timeout and SSO Session Max lifespan. The offline token is valid even after a user logout or server restart. However, by default the token will expire after 30 days of inactivity.
```

Comment:
Last sentence contradicts "token will never expire by default". Also should we spell out SSO? Is this referring to Red Hat single sign-on / Keycloak?

Funnily enough idle time out and max lifespan are Infinispan / Data Grid parameters that configure how long tokens are stored in the system (RHSSO relies on RHDG as a data store). I'm not sure we really need to explain that level of detail here. It's probably too lowlevel to mention the underlying implementation details. How about possibly condensing this down a little?

Offline tokens expire after 30 days of inactivity. You can periodically refresh offline tokens to prevent them from expiring, which means users need to request a new offline token. (Not sure about the 2nd part about requesting a new token but it might be worth explaining the benefit of preventing tokens from expiring.)

HTH.

Reply:
Hi @oraNod. Thanks for the comments/suggestions. This PR is a WIP so not quite ready for a peer-review just yet (sorry about the notification). I was troubleshooting an issue and had to create a PR to do that. 

Your comments and suggestions are valid. I'll revise/edit accordingly after the SME review :)

Reply:
Sorry for jumping the gun with the peer review @lmalivert Hope you have a great weekend.

Reply:
No problem :) Honest mistake. Thanks for adding the new label 👍🏽